SPEC.md

```diff
@@ -324,6 +324,13 @@ The test index makes use of symbolic language in describing connection and messa
     InvalidData errors to indicate that connection couldn't be established at all or timeout errors to indicate
     that data has not been received in timely manner.
 
+### ZG-PERFORMANCE-002
+
+    The node sheds or rejects connections when necessary.
```

Comment:
This makes it sound as a resistance test.

Reply:
Agree, this test could be resistance but as @Rqnsom  said, it has been classified to performance before. IMHO it should be left in this category to be consistent with other projects.